scripts/build-desktop.sh

```diff
@@ -79,11 +79,17 @@ function joinExistingPath() {
 STATUSREACTPATH="$(cd "$SCRIPTPATH" && cd '..' && pwd)"
 WORKFOLDER="$(joinExistingPath "$STATUSREACTPATH" 'StatusImPackage')"
 DEPLOYQTFNAME='linuxdeployqt-continuous-x86_64_20181215.AppImage'
-APPIMAGETOOLFNAME='appimagetool-x86_64_20181109.AppImage'
+APPIMAGETOOLFNAME='appimagetool-x86_64_20190221.AppImage'
 DEPLOYQT=$(joinPath . "$DEPLOYQTFNAME")
 APPIMAGETOOL=$(joinPath . "$APPIMAGETOOLFNAME")
 STATUSIM_APPIMAGE_ARCHIVE="StatusImAppImage_20181208.zip"
 
+APPIMAGE_OPTIONS=""
+if [[ ! -c /dev/fuse ]]; then # doesn't exist when run under docker
+    # We extract it to avoid having to use fuse and privileged mode in docker
+    APPIMAGE_OPTIONS="--appimage-extract-and-run"
```

Comment:
Isn't `--appimage-extract` enough here? I'm not familiar with `--appimage-extract-and-run`

Reply:
It's exactly the same as `--appimage-extract` except it uses a temporary directory and cleans up after itself.

Reply:
oh, TYL